components/ui/Navbar.js

```diff
@@ -20,6 +20,16 @@ const navbarItems = [
   }
 ];
 
+// NavItem component
+const NavItem = ({ title, link, isActive }) => {
```

Comment:
Here you've created the NavItem, but you didn't use it in the Navbar, how come? This was actually a nice idea!